src/main/java/org/zalando/nakadi/repository/MultiTimelineEventConsumer.java

```diff
@@ -307,6 +308,9 @@ void onTimelineChange(final String eventType) {
     public void close() throws IOException {
         try {
             reassign(Collections.emptySet());
+            for (final EventConsumer.LowLevelConsumer consumer : eventConsumers.values()) {
+                consumer.close();
+            }
```

Comment:
This is not needed:
`reassign() -> onTimelinesChanged() -> electTopicRepositories() -> close not used`

Reply:
why does it exist then?